app/repositories/SessionRepository.scala

```diff
@@ -41,6 +41,7 @@ object DefaultSessionRepository {
                                   processId: String,
                                   process: Process,
                                   labels: Map[String, Label],
+                                  flowStack: List[Flow],
```

Comment:
Should this be List[FlowStage] as presumably you will be saving instances of Flow and Continuation?

Reply:
Yes, fixed